Application/Test.Common/ModelBuilders/PropertyDescriptor.cs

```diff
@@ -0,0 +1,58 @@
+using System.Data;
+using EdFi.Ods.Common.Models.Domain;
+
+namespace Test.Common.ModelBuilders;
+
+public class PropertyDescriptor
+{
+    private readonly PropertyCollectionDescriptor _propertyCollectionDescriptor;
+
+    protected internal readonly string PropertyName;
+    protected internal PropertyType PropertyType;
+    protected internal bool IsIdentifying;
+    protected internal bool IsServerAssigned;
+
+    protected internal PropertyDescriptor(PropertyCollectionDescriptor propertyCollectionDescriptor, string name)
+    {
+        _propertyCollectionDescriptor = propertyCollectionDescriptor;
+        PropertyName = name;
+            
+        // By default, it's a string
+        AsString(20);
```

Comment:
Since 20 is the default value of the parameter, it is not needed here.

Reply:
I'll clean this one up too... probably another case of the signature of the called method evolving to include a default that matched my initial choice.